contracts/token/adapters/UniswapExample.sol

```diff
@@ -1,5 +1,5 @@
 // SPDX-License-Identifier: MIT
-pragma solidity 0.8.17;
+pragma solidity ^0.8.20;
 pragma abicoder v2;
```

Comment:
I think this has been by activated by default since 0.8.0

Reply:
Yup, removed it